osi_object.proto

```diff
@@ -826,4 +837,47 @@ message MovingObject
             }
         }
     }
+
+    //
+    // \brief Information for the classification of pedestrians regarding
+    // \c MovingObject (host or other).
+    //
+    message PedestrianClassification
+    {
+        // The head pose describes the pedestrian's head orientation. In ground truth / traffic update it is relative to
+        // the global ground truth frame. In sensordata (detected pedestrians) it is relative to the host vehicle frame.
```

Comment:
Comment Kmeid: Host vehicle frame correct or sensor frame?

Reply:
@kmeids the description for detected pedestrians currently is:
```
Describes the head orientation w.r.t. the host vehicle orientation.
```
Therefore, I wrote "host vehicle frame" to say the same thing and be backwards compatible. Nevertheless, I think sensor frame is better (i.e. physical mounting position / mounting position, depending on sensor technology). I guess we can make this change only in a major OSI version.